docs/setup/network-config.md

```diff
@@ -0,0 +1,60 @@
+# Network Recommendations
+This page details the recommendations for network settings to communicate with the Create® 3 via a network. 
+
+## Wi-Fi Networks
+There are several Wi-Fi configurations that must be in place to properly communicate with the Create® 3 via a Wi-Fi network. Below are the currently supported configurations. 
```

Comment:
Nitpick:
go to a new line after a full stop `.`

Reply:
Adding a single line break in the md after each sentence allows diffs to be cleaner, but the sentences will appear in the same paragraph in the rendered document. Please do this throughout this PR. (Personal note: I hate this because it makes it harder to read the md as plain text IMHO, but since it makes diffs easier, and most people will be reading the rendered version, we have adopted this style.)